python/tests/aggregate_tests/test_row_count_col.py

```diff
@@ -0,0 +1,58 @@
+from .aggtst_base import TstView
+
+
+class aggtst_row_count_col(TstView):
```

Comment:
these do not seem to return empty sets, do they belong in this PR?

Reply:
Only COUNT was left for ROW type so I created a single PR for them and empty sets.

Reply:
but do we have tests for ROW where the result is empty?